wallet/src/model.rs

```diff
@@ -218,44 +219,6 @@ pub struct Transactions {
     pub total: u32,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
-pub struct OutPtr {
```

Comment:
Removing this may be a breaking change, depending on the layout of the new type in the database